src/lib/graphql/skylark/dynamicQueries.ts

```diff
@@ -351,3 +355,54 @@ export const createSearchObjectsQuery = (
 
   return gql(graphQLQuery);
 };
+
+export const createGetObjectRelationshipsQuery = (
+  object: SkylarkObjectMeta | null,
+  relationshipsFields: { [key: string]: NormalizedObjectField[] },
+) => {
+  if (!object || !object.operations.get) {
+    return null;
+  }
+
+  const query = {
+    query: {
+      __name: createGetObjectRelationshipsQueryName(object.name),
+      __variables: {
+        uid: "String",
+        nextToken: "String",
+        ...common.variables,
+      },
+      getObjectRelationships: {
+        __aliasFor: object.operations.get.name,
+        __args: {
+          ...common.args,
+          uid: new VariableType("uid"),
+        },
+        ...object.relationships.reduce((acc, currentValue) => {
+          return {
+            ...acc,
+            [currentValue.relationshipName]: {
+              __args: {
+                limit: 50,
+                next_token: new VariableType("nextToken"),
```

Comment:
only note in this file is we will likely want to have the variable called 
```
new VariableType(`${currentValue.relationshipName}NextToken`),
```
but we could change it later